addons/hr_expense/models/hr_expense.py

```diff
@@ -39,7 +39,7 @@ class HrExpense(models.Model):
                               ('refused', 'Refused')
         ], compute='_compute_state', default="draft", string='Status', index=True, readonly=True, copy=False, required=True, store=True,
         help="Status of the expense.")
-    sheet_id = fields.Many2one('hr.expense.sheet', string="Expense Report", readonly=True)
+    sheet_id = fields.Many2one('hr.expense.sheet', string="Expense Report", readonly=True, copy=False)
```

Comment:
You could order your parameter alphabetically. You are the kind of person who wear blue socks with red baskets.